.gitignore

```diff
@@ -1,3 +1,4 @@
+.cache
```

Comment:
Will the cache file show up here (at the package root) when scons runs pytest? In the past I think it may have showed up in `tests/.cache`